lib/services/role.go

```diff
@@ -1043,6 +1051,13 @@ func (set RoleSet) CheckKubeGroupsAndUsers(ttl time.Duration, overrideTTL bool)
 		}
 	}
 	for _, role := range set {
+		ok, err := RoleMatchers(matchers).MatchAll(role, types.Deny)
```

Comment:
Should we use `MatchAny` instead of `MatchAll` when checking Deny rules?

Reply:
Right, Fixed. Thanks.